plugins/examples/clusterbuttonoverride/src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Headlamp, Plugin, Registry } from '@kinvolk/headlamp-plugin/lib';
+import React from 'react';
+// import { SectionBox } from '@kinvolk/headlamp-plugin/lib/CommonComponents';
+// import { K8s } from '@kinvolk/headlamp-plugin/lib/K8s';
+// import { Typography } from '@material-ui/core';
+
+class ClusterChooserButtonOverridePlugin extends Plugin {
+  initialize(registry: Registry) {
+    console.log('clusterbuttonoverride initialized');
+
+    // Register your plugin methods here, like:
+    // registry.registerSidebarItem(...);
+    // registry.registerRoute(...);
```

Comment:
Remove these and other comments, as this is an example plugin for a particular purpose, not a generic one.

Reply:
done